Dockerfile

```diff
@@ -16,7 +16,6 @@ RUN pip install -r requirements.txt
 RUN cd /workspace && \
     git clone https://github.com/codait/deep-histopath && \
     cd deep-histopath && \
-    git checkout e0c93c3543830a0ee07ad9408e844465f045781c && \
```

Comment:
I like the idea of locking this to a specific commit or maybe a release so that a changed to the deep histopath master branch doesn't break this model. Maybe you can just change this to the current latest commit?

Reply:
Yeah, that is a good idea. I just revert it and update the commit id.